net/local/local_recvmsg.c

```diff
@@ -108,7 +108,11 @@ static int psock_fifo_read(FAR struct socket *psock, FAR void *buf,
         }
       else
         {
-          nerr("ERROR: Failed to read packet: %d\n", ret);
+          if (ret != -EAGAIN)
+            {
+              nerr("ERROR: Failed to read packet: %d\n", ret);
+            }
+
```

Comment:
@zhhyu7 maybe instead of hiding the -EAGAIN error you could transform it in a warning or even an info, maybe in some case the -EAGAIN could indicate some minor issues in the application itself.